src/styles/amino.css

```diff
@@ -110,6 +110,7 @@ h6 {
 
 .toasts-wrapper {
   position: fixed;
+  z-index: 9999999;
```

Comment:
What is the Z-index you are building over?
9999999 seems excessive.

Reply:
> `z-index: 990;`

Is the z-index set for `CoverSheet.` 

> > z-index: 999999;

is the z-index set for the `Toast` itself. 

I do agree that it is excessive. I will look into standardizing z-indices in a future update.

Reply:
I do like clear enumerations like [this suggestion](https://stackoverflow.com/a/13591180)